configs/BROTHERHOBBYF722V3/config.h

```diff
@@ -0,0 +1,117 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32F7X2
+#define BOARD_NAME        BHF722V3
+#define MANUFACTURER_ID   BROT
+
+#define USE_ACC
+#define USE_ACC_SPI_ICM42688P
+#define USE_GYRO
+#define USE_GYRO_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_BMP280
+#define USE_BARO_MS5611
+#define USE_BARO_DPS310
+#define USE_FLASH
+#define USE_FLASH_W25N02K
+#define USE_MAX7456
+
+#define BEEPER_PIN           PC3
+#define MOTOR1_PIN           PB0
+#define MOTOR2_PIN           PB1
+#define MOTOR3_PIN           PB10
+#define MOTOR4_PIN           PB11
+#define MOTOR5_PIN           PB4
+#define MOTOR6_PIN           PB5
+#define MOTOR7_PIN           PC8
+#define MOTOR8_PIN           PC9
+#define RX_PPM_PIN           PA3
+#define LED_STRIP_PIN        PA15
+#define UART1_TX_PIN         PA9
+#define UART2_TX_PIN         PA2
+#define UART3_TX_PIN         PC10
+#define UART4_TX_PIN         PA0
+#define UART5_TX_PIN         PC12
+#define UART6_TX_PIN         PC6
+#define UART1_RX_PIN         PA10
+#define UART2_RX_PIN         PA3
+#define UART3_RX_PIN         PC11
+#define UART4_RX_PIN         PA1
+#define UART5_RX_PIN         PD2
+#define UART6_RX_PIN         PC7
+#define I2C1_SCL_PIN         PB8
+#define I2C1_SDA_PIN         PB9
+#define LED0_PIN             PA8
+#define SPI1_SCK_PIN         PA5
+#define SPI2_SCK_PIN         PB13
+#define SPI1_SDI_PIN         PA6
+#define SPI2_SDI_PIN         PB14
+#define SPI1_SDO_PIN         PA7
+#define SPI2_SDO_PIN         PB15
+#define CAMERA_CONTROL_PIN   PB7
+#define ADC_VBAT_PIN         PC2
+#define ADC_RSSI_PIN         PC0
+#define ADC_CURR_PIN         PC1
+#define FLASH_CS_PIN         PB6
+#define MAX7456_SPI_CS_PIN   PB12
+#define GYRO_1_EXTI_PIN      PC4
+#define GYRO_1_CS_PIN        PA4
+
+#define TIMER_PIN_MAPPING \

```

Comment:
```#define TIMER_PIN_MAPPING \
        TIMER_PIN_MAP(0,  PB6,  1, -1)\ Delete
        TIMER_PIN_MAP(1,  PB7,  1,  0)\
        TIMER_PIN_MAP(2,  PB8,  1,  0)\ Delete
        TIMER_PIN_MAP(3,  PB9,  1, -1)\ Delete
        TIMER_PIN_MAP(4,  PB0,  2,  0)\
        TIMER_PIN_MAP(5,  PB1,  2,  0)\
        TIMER_PIN_MAP(6,  PB5,  1,  0)\
        TIMER_PIN_MAP(7,  PB4,  1,  0)\
        TIMER_PIN_MAP(8,  PA8,  1,  0)\ Delete
        TIMER_PIN_MAP(9,  PA3,  2,  0)\ Delete
        TIMER_PIN_MAP(10, PB3,  1,  0)\ This pin isn't even assigned. Delete
        TIMER_PIN_MAP(11, PC9,  2,  0)\
        TIMER_PIN_MAP(12, PB10, 1,  0)\
        TIMER_PIN_MAP(13, PB11, 1,  0)\
        TIMER_PIN_MAP(14, PC8,  2,  0)\
        TIMER_PIN_MAP(15, PA15, 1,  0) LED_STRIP won't work because it shares TIM2 with Motors 3 and 4. Needs to be on a different timer.
```